src/game/default/g_combat.c

```diff
@@ -137,21 +137,28 @@ static int16_t G_CheckArmor(g_entity_t *ent, const vec3_t pos, const vec3_t norm
 	const int16_t quantity = ent->client->locals.inventory[ITEM_INDEX(armor)];
 
 	vec_t protection = 0.0;
+	vec_t waste = 1.0;
 
-	switch (armor->tag) {
+	// energy weaps cut through armor more
+	int16_t type = (dflags & DMG_ENERGY) ? armor->tag-1 : armor->tag;
+
+	switch (type) {
 		case ARMOR_BODY:
-			protection = 1.0;
+			protection = 0.75;
+			waste = 0.65;
 			break;
 		case ARMOR_COMBAT:
 			protection = 0.66;
+			waste = 0.75;
 			break;
 		case ARMOR_JACKET:
-			protection = 0.5;
+			protection = 0.33;
+			waste = 0.85;
```

Comment:
Shouldn't the efficiencies be > 1.0, tho? If I'm reading this correctly, for every 1 damage that body armor saves you, it only uses .65 units of armor. I would think it makes more sense for 1 unit in damage saved should cost 1.33 (for example -- just throwing that out there) units of armor.